test/forms/TradingNameFormSpec.scala

```diff
@@ -29,7 +29,7 @@ class TradingNameFormSpec extends TestUtil {
     val unchangedErrorMessage: String = "captureTradingName.error.notChanged"
     val specialCharsErrorMessage: String = "captureTradingName.error.containsSpecialCharacters"
 
-    val testTradingName = "Valid` & Trading-' Name."
+    val testTradingName = "Valid & Trading' Name.-'’‘()<>!«»?+=%#*&$€£_@¥.,:;"
```

Comment:
There are some special chars allowed by the regex that are missing here. Given the issues we had with it do you think it's worth putting them all in and verifying all is good?

Characters missing: `{` `}` `"` `/` `\`